hack/cve-triage/cve_triage_core.py

```
@@ -0,0 +1,2091 @@
+        print(f"  ✅ Go CVE confirmed: {go_id}")
+        print(f"     Packages:  {', '.join(go_data['packages'])}")
+        print(f"     Fixed:     {go_data.get('fixed', 'unknown')}")
+        sym_count = sum(len(sg["names"]) for sg in go_data.get("symbols", []))
+        print(f"     Symbols:   {sym_count} vulnerable symbol(s)")
+    else:
+        if not is_go_jira:
+            print(f"\n❌ {cve_id} is not a Go CVE (neither Jira nor OSV indicate Go).")
+            print("   This script handles Go CVEs only. Manual review needed.")
+            return 1
+        print(f"\n  ⚠️  {cve_id} is not yet in the Go vulnerability DB.")
+        print(f"     govulncheck cannot scan for it — findings are indexed by GO-* IDs.")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)